skyportal/handlers/api/analysis.py

```diff
@@ -1392,6 +1392,7 @@ async def get(
                             await self.send_file(
                                 output_data, filename, output_type=output_type
                             )
+                            return
```

Comment:
This return statement seems out of place...

Reply:
It's not. It sends the file and returns.

Reply:
Shouldn't this be a `retutn.success`?

Reply:
That returns json, which is not what we want.

Reply:
I don't think we are supposed to have a handler that doesn't return either success or failure.

Reply:
I think the file download stuff is an exception here.

Reply:
So if `send_file` replaces this action of passing back `success`, what happens in other cases where there isn't this `return` statement right after it?

Reply:
ok I see how it works. This is fine